week-2/extended-project/banzai/todo-list/cli.rb

```diff
@@ -0,0 +1,35 @@
+# define global variable
+$bd_name = File.expand_path('../', __FILE__) + '/bd.txt' 
```

Comment:
@condef5 Can you explain me what this line does?

Reply:
This line load the absolute directory and join with name file, in this case `bd.txt`

Reply:
Thanks :+1: